.stickler.yml

```diff
@@ -0,0 +1,15 @@
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    # indicate where is the config file for stylelint
+    config: './.eslintrc.json'
+
+# add the files here you want to be ignored by stylelint
+files:
+  ignore:
+      - 'dist/*'
+
+# PLEASE DO NOT enable auto fixing options
+# if you need extra support from you linter - do it in your local env as described in README for this config
+
+# find full documentation here: https://stickler-ci.com/docs
```

Comment:
- [ ] Recently Microverse change the PR linter solution from the Stickler to Github Action. The Github Actions was properly set on your repository as requested, so there is no need to use the Stickler anymore. Please delete the `.stickler.yml` file

Reply:
We had no .stickler.yml but it requested in order to reach 100% for the pull request approval. I think it was because Martin has turned on the stickler on the webpage that activates it. Thank you.